packages/quantic/force-app/main/default/lwc/exampleStandaloneSearchbox/exampleStandaloneSearchbox.css

```diff
@@ -0,0 +1,10 @@
+.page-content {
+    max-width: 1400px;
+    width: 100%;
+    min-width: 800px;
+    min-height: 500px;
```

Comment:
Do we have some css var for those magic numbers ?

Reply:
Not for this, it's not really meant to be "used" in a community or anything. It's mainly for us to be able to show the standalone searchbox in localdev server.

Otherwise if you just put the standalone the whole content area of the localdev is only as big as the searchbox input which means we get a scrollbar to see query suggestions, etc.

So I made that one kinda like the exampleSearch. So I wouldn't worry too much about having this "scale" or be editable easily anything like that, what do you think?

I was unsure even to include this component in the PR.